src/main/java/com/ericsson/ei/handlers/MongoDBHandler.java

```diff
@@ -342,4 +342,19 @@ public void dropDatabase(String databaseName) {
         MongoDatabase db = mongoClient.getDatabase(databaseName);
         db.drop();
     }
+
+    /**
+     * Returns a boolean indicating that if MongoDB is up and running or not.
+     *
+     * @return
+     */
+    public boolean isMongoDBServerUp() {
+        try {
+            mongoClient.getAddress();
```

Comment:
have you manually tested if the address is not set when you take down Mongo?

Reply:
It will throw exception if MongoDB is down.